### ISSUE TITLE
How to build docker container

### ISSUE THREAD

**Issue by amaline** [spring-cloud/spring-cloud-dataflow-metrics-collector#60]
Is there documentation on how to rebuild the docker container.  Current container is not ARM compatible.

**Comment by sabbyanandan** [comment]
Hi, @amaline. Thank you for reporting this. I'm guessing by "Current container", you're referring to the Docker images that we publish and the fact that it is not ARM compliant. Did I get that right?

**Comment by amaline** [comment]
Yes, I did figure out how to rebuild the container (just had to read the README a bit more carefully).  Ran into the issue that the base Java image also does not have mult-architecture support (it is also deprecated).  Built a new image based on OpenJDK and then discovered that OpenJDK does not have an ARM just in time compiler which caused the scdf server to take over an hour to startup.

- upon further search I just found https://hub.docker.com/r/arm64v8/openjdk/ - I will give that a try next week.
